packages/website/plugins/generated-rule-docs/insertions/insertBaseRuleReferences.ts

```diff
@@ -27,15 +30,57 @@ export function insertBaseRuleReferences(page: RuleDocsPage): string {
     page.headingIndices.howToUse + 1,
     0,
     {
-      lang: 'js',
-      type: 'code',
-      meta: 'title=".eslintrc.cjs"',
-      value: `module.exports = ${getEslintrcString(
-        extendsBaseRuleName,
-        page.file.stem,
-        true,
-      )};`,
-    } as mdast.Code,
+      type: 'mdxJsxFlowElement',
+      name: 'Tabs',
+      children: [
+        {
+          type: 'mdxJsxFlowElement',
+          name: 'TabItem',
+          attributes: [
+            {
+              type: 'mdxJsxAttribute',
+              name: 'value',
+              value: 'Flat Config',
+            },
+          ],
+          children: [
+            {
+              type: 'code',
+              lang: 'js',
+              meta: 'title="eslint.config.mjs"',
+              value: `export default tseslint.config(${getEslintConfigString(
+                extendsBaseRuleName,
+                page.file.stem,
+                true,
+              )})`,
```

Comment:
Missing a `;` to be consistent with the legacy configs & rest of the site:

```suggestion
              )});`,
```

Reply:
I fixed it in [c2aad1c](https://github.com/typescript-eslint/typescript-eslint/pull/10164/commits/c2aad1c8b0c321cb51ae4a31dd0d91b96a0a8beb).